pootle/apps/pootle_misc/checks.py

```diff
@@ -1008,13 +1010,17 @@ def get_qualitycheck_schema(path_obj=None):
         Category.COSMETIC: _("Cosmetic"),
         Category.FUNCTIONAL: _("Functional"),
         Category.EXTRACTION: _("Extraction"),
-        Category.NO_CATEGORY: _("Other"),
+        Category.NO_CATEGORY: _("Other")
```

Comment:
Please do not remove the trailing comma. If any new entry is added in the future this would save us from having an unnecessary extra diff line.